usort/tests/util.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import unittest
+
+import libcst as cst
+
+from ..util import parse_import
+
+
+class UtilTest(unittest.TestCase):
+    def test_parse_import(self) -> None:
+        with self.subTest("simple import"):
```

Comment:
Not sure you're gaining anything (even vertical space) by using `subTest`, and you're losing some horizontal space.  If there is a reason (variables reused in a way I'm not seeing), that's subtle and deserves a comment.

Reply:
Primarily was done to have a single test function, but get feedback on multiple conditions. Would you prefer them to each be a separate test function?